web/ui/src/features/widgets/Table.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+
+import styles from './Table.module.css';
+
+interface Props {
+  tableHeaders: string[];
+  style?: React.CSSProperties;
+  renderTableData: () => JSX.Element[];
+}
+
+/**
+ * Simple table component that accept a custom header, and custom render
+ * function for the table data
+ */
+const Table = ({ tableHeaders, style = {}, renderTableData }: Props) => {
+  return (
+    <table className={styles.table}>
+      <col span={1} style={style} />
```

Comment:
I believe this is throwing an error
![Screen Shot 2023-03-07 at 2 27 13 PM](https://user-images.githubusercontent.com/114438185/223576575-481c9581-0918-411c-81f7-d730338f9de4.png)

Reply:
Fixed using `<colgroup>`